lib/src/install/baseline.rs

```diff
@@ -73,9 +74,14 @@ pub(crate) struct InstallBlockDeviceOpts {
     ///
     /// direct: Filesystem written directly to block device
     /// tpm2-luks: Bind unlock of filesystem to presence of the default tpm2 device.
+    /// password-luks: Encrypt filesystem with LUKS using a password.
     #[clap(long, value_enum)]
     pub(crate) block_setup: Option<BlockSetup>,
 
+    /// Password for LUKS encryption when `block_setup` is `password-luks`.
+    #[clap(long)]
+    pub(crate) luks_password: Option<String>,
```

Comment:
The reason `cryptsetup` reads from stdin (as opposed to having a parameter) is because command line arguments can be globally visible on Unix. It's probably simplest for us to have `--luks-password-file` that reads from a file say. (And when we do that we should sanity check the file isn't world readable either)